apps/frontend/Gump/components/recipe/RecipeFooter.vue

```diff
@@ -54,7 +54,7 @@ async function saveClick() {
 <template>
   <div v-if="props.recipe" flex="~ row" mx-2 justify-between text-left font-mono text-xl>
     <div flex="~ row" items-center>
-      <div i-fa6-solid-eye orangeIcon />
+      <div orangeicon i-fa6-solid-eye />

```

Comment:
itt szépen elcseszted az icont (nagybetű kell legyen)

Reply:
upszika, nem vettem észre hogy azt is átírta, csak izélt, hogy rossz a sorrend